data/presets/amenity/parcel_locker.json

```
@@ -1,4 +1,7 @@
 {
+    "aliases": [
+        "package locker"
```

Unlike terms, aliases are Title Cased just like the preset name.

```suggestion
        "Package Locker"
```

---

Ah thanks for the heads up, should be fixed now.